asset/tests/system/test_vpcsc.py

```diff
@@ -36,7 +36,7 @@ def _is_rejected(call):
         try:
             responses = call()
         except exceptions.PermissionDenied as e:
-            return e.message == "Request is prohibited by organization's policy"
+            return "Request is prohibited by organization's policy" in e.message
```

Comment:
The assert calling this function [failed on python 3.7](https://source.cloud.google.com/results/invocations/191cc138-d1cc-4d8f-a71e-79bc28332ba5/targets/cloud-devrel%2Fclient-libraries%2Fgoogle-cloud-python%2Fpresubmit%2Fasset/log).

Reply:
I'm not sure how this is supposed to be exercised under a non-VPCSC setup.

Reply:
For instance, running on `master` locally with the "outside project" set, and after configuring the CI project to allow the Asset API, both system tests fail for me.